src/pages/Admin/Home/index.tsx

```diff
@@ -280,6 +280,8 @@ type AdminVoucherModalProps = Omit<ModalProps, "children" | "isOpen"> & {
   type?: types | null
 }
 
+var uploadDisabled = false;
```

Comment:
And also can you add an `is` prefix like `isUploadDisabled`. For Boolean variables, it is advisable to prefix the name with an `is` to make it easier for others to know that it is a Boolean variable.